src/main/java/pickleib/web/utilities/WebUtilities.java

```diff
@@ -357,4 +354,41 @@ public void fillIframeForm(
         }
         driver.switchTo().parentFrame();
     }
+
+
+    /**
+     *
+     * Verify that the attribute {attribute name} present for the {element name}
+     *
+     * @param element target element
+     * @param attributeName target attribute name
+     */
+    public boolean attributeNamePresent(String attributeName, WebElement element) {
+        long initialTime = System.currentTimeMillis();
+        String caughtException = null;
+        int counter = 0;
+        do {
+            try {
+                Set<String> elementAttributes = getElementJson(element).getAsJsonObject().asMap().keySet();
+                for (String attribute : elementAttributes) return attribute.equals(attributeName);
```

Comment:
The method name suggests that the method is supposed to check if an attribute is present, however the returned value is whether this attribute ia equal to certain other value.. 

Please resolve this discrepancy.